src/lib/user/user-models.ts

```diff
@@ -46,3 +46,17 @@ export type MyDogDetails = StatusSet & {
 
   dogReports: MyDogReport[];
 };
+
+export type MyAccount = {
+  userId: string;
+  userCreationTime: Date;
+  userName: string;
+  userResidency: string;
+  userEmail: string;
+  userPhoneNumber: string;
+};
+
+export type MyLastContactedTime = {
+  userId: string;
+  userLastContactedTime: Date;
```

Comment:
This is actually nullable. (The SQL query returns NULL when there are no `calls` related to the user.)

Reply:
Done! b07f27143d693cce888d59d16c08072d117813f4